cmd/zetaclientd/start.go

```diff
@@ -113,15 +115,15 @@ func start(_ *cobra.Command, _ []string) error {
 	startLogger.Debug().Msgf("CreateAuthzSigner is ready")
 
 	// Initialize core parameters from zetacore
-	err = zetaBridge.UpdateConfigFromCore(cfg, true)
+	appContext := appcontext.NewAppContext(corecontext.NewZetaCoreContext(cfg), cfg, masterLogger)
```

Comment:
Would it be simpler if we can put `appcontext` & `corecontext` source files in one single `context` package. The code will look like
`appContext := context.NewAppContext(context.NewZetaCoreContext(cfg), cfg, masterLogger)`

Reply:
okay. We can leave them separate packages and see.